packages/client/src/app/components/modals/party/kamis/expanded/StatusDisplay.tsx

```
@@ -0,0 +1,139 @@
+    if (isHarvesting(kami)) return () => selectNode(kami.harvest?.node?.index!);
+  };
+
+  /////////////////
+  // RENDER
+
+  return (
+    <Container>
+      {header && <Text size={0.75}>{header}</Text>}
+      <Description onClick={getDescriptionOnClick(kami)}>
+        {description.map((text) => (
+          <Text size={0.6}>{text}</Text>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)